### ISSUE TITLE
[ToggleGroup] Doesn't communicate group to SRs

### ISSUE THREAD

**Issue by jjenzz** [radix-ui/primitives#754]
When you reach the toggle **group** with VoiceOver (haven't tested NVDA but should be checked too) it will tell you that you are on a toggle button but it isn't clear it is a group with others in the list. Because of this, as an SR user it's not obvious I can use my arrow keys to navigate the list. I end up hitting tab and completely bypassing the other items.

**Comment by benoitgrelard** [comment]
Interesting, where are you trying that? Perhaps that example doesn't have a label?
We have `role="group"` on it, and our stories have `aria-label` on the group too.

**Comment by jjenzz** [comment]
We have `role=group` but focus never lands there because of the `RovingFocusGroup` proxying.

**Comment by jjenzz** [comment]
Ah, I'm testing in the docs and there is no `aria-label` on the group. Perhaps we need to make the `aria-label` or `aria-labelledby` stuff required somehow so people don't accidentally make it inaccessible.

With the label it just reads "Toggle button, Test, Group". It doesn't communicate how many items there are but dunno if that matters.

**Comment by jjenzz** [comment]
This is the markup for the [text alignment toggle group](https://www.w3.org/TR/wai-aria-practices-1.1/examples/toolbar/toolbar.html) in the WAI ARIA docs:

![image](https://user-images.githubusercontent.com/175330/124130676-f10ba380-da76-11eb-99e0-93c68766c68c.png)

With this markup it's really obvious it's a list... maybe we should do the same?

**Comment by benoitgrelard** [comment]
Yeah, it's difficult to make this stuff required though that's the problem, because there are multiple ways of labelling.

**Comment by benoitgrelard** [comment]
Maybe it's a matter of generalising warnings like we do in `Dialog` and `AlertDialog`?

**Comment by jjenzz** [comment]
Yeah, that's probably the best solution :+1:
